sds_data_manager/stacks/dynamodb_stack.py

```diff
@@ -0,0 +1,100 @@
+# Create stack for DynamoDB
+from aws_cdk import Environment, RemovalPolicy, Stack
+from aws_cdk import aws_dynamodb as dynamodb
+from constructs import Construct
+
+
+class DynamoDB(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        sds_id: str,
+        table_name: str,
+        partition_key: str,
+        sort_key: str,
+        env: Environment,
+        on_demand: bool = True,
+        read_capacity: int = None,
+        write_capacity: int = None,
+        **kwargs,
+    ):
+        super().__init__(scope, construct_id, **kwargs)
```

Comment:
It should be:
super().__init__(scope, construct_id, env=env, **kwargs)

Reply:
I don't use env. should I still set env for future use?

Reply:
I'd actually just remove it from your listed keyword arguments above, and then `**kwargs` will forward it automatically if you pass in `env=env` during creation, but won't require it here because you don't actually use it below.